### PR TITLE
[MB-293][MB-284][MB-305][MB-301][MB-297] Delete unused reducers

### PR DESCRIPTION
**Pull request by suzubara**
## Description

This PR completes a major portion of the Redux Refactor project by deleting reducers that have been deprecated and are no longer referenced in the code. It also removes redundant update actions that were being called in the entities saga (these were kept around to make sure API data was updating both the new `entities` reducer and the legacy reducers).

## Reviewer Notes

No user-facing changes expected with this PR. As long as tests pass and critical UX flows look fine it should be good to go!

## Setup

To run the app locally:

```sh
make server_run
make client_run
```

## Code Review Verification Steps

* [ ] If the change is risky, it has been tested in experimental before merging.
* [ ] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#logging)
* [ ] The requirements listed in [Querying the Database Safely](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#querying-the-database-safely) have been satisfied.
* Any new migrations/schema changes:
  * [ ] Follow our guidelines for zero-downtime deploys (see [Zero-Downtime Deploys](https://github.com/transcom/mymove/wiki/migrate-the-database#zero-downtime-migrations))
  * [ ] Have been communicated to #g-database
  * [ ] Secure migrations have been tested following the instructions in our [docs](https://github.com/transcom/mymove/wiki/migrate-the-database#secure-migrations)
* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* Jira stories for this change: [PPM reducer](https://dp3.atlassian.net/browse/MB-293), [moves reducer](https://dp3.atlassian.net/browse/MB-284), [legalese reducer](https://dp3.atlassian.net/browse/MB-305), [orders reducer](https://dp3.atlassian.net/browse/MB-301), [service member reducer](https://dp3.atlassian.net/browse/MB-297)

### CODE REVIEW

File: src/App/index.jsx [hunk 1]
```diff
@@ -12,6 +12,7 @@ import { AppContext, defaultOfficeContext, defaultMyMoveContext, defaultAdminCon
 import { detectFlags } from 'shared/featureFlags';
 
 import '../icons';
+import 'shared/shared.css';
```
Comment (suzubara) [inline src/App/index.jsx:15]: I believe moving this import out of `utils.js` helped to resolve the mini CSS extract build errors we were seeing in CI @duncan-truss

**Comment by suzubara** [comment]
I ran into a CSS build error in CI and was debugging that, which is why I've also removed the `custom.scss` import from CSS module files (this is not needed)

File: src/scenes/Review/EditOrders.jsx [hunk 2]
```diff
@@ -239,7 +239,6 @@ function mapStateToProps(state) {
     existingUploads: uploads,
     error: get(state, 'orders.error'),
     formValues: getFormValues(editOrdersFormName)(state),
-    hasSubmitError: get(state, 'orders.hasSubmitError'),
```
Comment (suzubara) [inline src/scenes/Review/EditOrders.jsx:242]: this prop is not referenced or used in this component

File: src/App/index.jsx (hunk 1 above)
Comment (duncan-truss) [reply src/App/index.jsx:15]: Oh nice you started seeing those errors in CI too?  I started with the CSS module non-required imports like customs and mixins too but it was really the import order in the parent components it seemed that fixed what was on my branch.

**Review by duncan-truss — APPROVED** [review]
I think even the `src/shared/styles/_mixins.scss` is only for a few custom mixins we have (I think the underline one was for a navbar) and is not required if you're only using the standard uswds mixins.

**Review by amanda5iu — APPROVED** [review]

**Review by YanZ777 — APPROVED** [review]
Looks good to me!

**Pull request merged**